Replace instructions to modify pubspec.yaml with `flutter pub add` command in multiple pages (#8950)
flutter/website: src/cookbook/navigation/set-up-universal-links.md

    $ flutter create deeplink_cookbook
    ```

> 2. To include `go_router` package in your app, run `flutter pub add`:

Reviewer: ```suggestion
2. To include the `go_router` package as a dependency, 
   run `flutter pub add`:
```
